cmake/projects/libxml2/schemes/url_sha1_libxml2_msvc.cmake.in

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2016, Ruslan Baratov
+# Copyright (c) 2017, Sacha Refshauge
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+# Scheme for download and install LibXML2 library for Windows
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+include(hunter_libxml2_install_tools)
+include(hunter_fatal_error)
+include(hunter_internal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+
+hunter_status_debug("Scheme: url_sha1_libxml2_msvc")
+
+# Check preconditions
+hunter_test_string_not_empty("@HUNTER_SELF@")
+hunter_test_string_not_empty("@HUNTER_EP_NAME@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_URL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SHA1@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_DOWNLOAD_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_SOURCE_DIR@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_INSTALL_PREFIX@")
+hunter_test_string_not_empty("@HUNTER_MSVC_ARCH@")
+hunter_test_string_not_empty("@HUNTER_MSVC_VCVARSALL@")
+hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_DIR@")
+hunter_test_string_not_empty("@HUNTER_GLOBAL_SCRIPT_DIR@")
+
+if (BUILD_SHARED_LIBS)
+  set(shared "static=no")
+else()
+  set(shared "static=yes")
+endif()
+
+# Use native slashes to help nmake if statements work
+file(TO_NATIVE_PATH "@HUNTER_PACKAGE_INSTALL_PREFIX@" XML_INSTALL_DIR)
+
+ExternalProject_Add(
+    "@HUNTER_EP_NAME@"
+    URL
+    "@HUNTER_PACKAGE_URL@"
+    URL_HASH
+    SHA1=@HUNTER_PACKAGE_SHA1@
+    DOWNLOAD_DIR
+    "@HUNTER_PACKAGE_DOWNLOAD_DIR@"
+    SOURCE_DIR
+    "@HUNTER_PACKAGE_SOURCE_DIR@"
+    INSTALL_DIR
+    "@HUNTER_PACKAGE_INSTALL_PREFIX@"
+        # not used, just avoid creating Install/<name> empty directory
+    CONFIGURE_COMMAND
+    "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@" &&
```

Comment:
Do not use `&&`, see https://github.com/ruslo/hunter/issues/246

Reply:
You can create a script and put all stuff there. See Boost example:
* https://github.com/ruslo/hunter/blob/929ffc26701dab21c2468b2c199a0df1010432e7/cmake/projects/Boost/schemes/url_sha1_boost.cmake.in#L53-L60
* https://github.com/ruslo/hunter/blob/929ffc26701dab21c2468b2c199a0df1010432e7/cmake/projects/Boost/scripts/patched_boostrap.bat.in#L1-L3